webapi/src/main/scala/org/knora/webapi/KnoraService.scala

```diff
@@ -241,20 +410,31 @@ trait KnoraService {
         if (jaegerReporter) {
             Kamon.addReporter(new JaegerReporter()) // tracing
         }
-
-        Http().bindAndHandle(Route.handlerFlow(apiRoutes), settings.internalKnoraApiHost, settings.internalKnoraApiPort)
-        println("")
-        println("----------------------------------------------------------------")
-        println(s"Knora API Server started at http://${settings.internalKnoraApiHost}:${settings.internalKnoraApiPort}")
-        println("----------------------------------------------------------------")
     }
 
     /**
-      * Stops Knora.
+      * Checks if application state is 'Running' and only then returns.
       */
-    def stopService(): Unit = {
-        system.terminate()
-        CacheUtil.removeAllCaches()
-        //Kamon.shutdown()
+    private def waitForRunningState(): Unit = {
```

Comment:
This method is never called, can it be deleted?

Reply:
yes, forgot to remove it.